### ISSUE TITLE
Animate the refresh active icon

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-client#343]
## Description

When a user clicks on the refresh icon, we display the [refresh_active](https://github.com/freedomofpress/securedrop-client/blob/master/securedrop_client/resources/images/refresh_active.svg) icon. This should be animated (["rotating at a pleasingly elegant pace... nothing frenetic or sluggy"](https://github.com/freedomofpress/securedrop-ux/issues/17#issuecomment-472164510))

**Comment by eloquence** [comment]
Before we can consider this dev-ready, I think we should resolve the following questions:

1) Do we want to [keep user-initiated syncs around at all](https://github.com/freedomofpress/securedrop-client/issues/655#issuecomment-566744750)?

2) What events should trigger this animation? Only user syncs (provided we keep them), or background syncs as well? How about user-initiated downloads or other network operations? Note that downloads currently add a "Downloading file" status update to this same area of the client, so we're already overloading it semantically beyond syncs.

3) For a sync, should the animation only last for the duration of the initial "is there anything new" check, or until all messages and replies associated with it have been downloaded?

Back in July, Nina created [this spreadsheet](https://docs.google.com/spreadsheets/d/1jXalueqcT8RWWG7osGLFO8dsG6_EB9NdbfJfE_NEmqw/edit#gid=0) which conveys a set of proposals for how the icon and status message should be updated based on user actions, and which proposes answers to some (but not all) of these questions.

For the beta, I would encourage us to look for ways to dramatically reduce complexity here. Here's a proposal:
- Eliminate user-initated syncs entirely.
- Animate the icon during syncs, including any message/reply downloads associated with a sync.
- Don't animate the icon or update the status for _other_ user-initiated actions. Instead, show such operations in context as we've already started doing with replies.

In other words, treat it as the "sync progress area", not the "all network activity reported here" area. Even in that context, we might want to _not_ animate the icon during the "is there anything new" stage of the sync, if we perform this check constantly, because that could get very visually annoying and distracting.

**Comment by eloquence** [comment]
@ninavizz and I discussed the approach outlined above today. If we go down this road, it may make sense to change to a different icon entirely -- the current icon selection very much reflects the intention of an _interactive_ refresh. 

For an operation that frequently runs in the background, something like a color indicator, a pulse loader, or a subtle spinner may be a better fit. And we'd still want to decide what _stages_ of the operation to animate it for: just the metadata sync, just the message/reply downloads, or both.

**Comment by redshiftzero** [comment]
#655 is closed so this one will get some broader changes since the user will no longer click on the refresh icon in order to trigger a sync

**Comment by ninavizz** [comment]
OH!! I think w can close this—or shd it be a new issue for Animate The Sync Icon?

**Comment by ninavizz** [comment]
Clozing, cuz there ain't no more refresh button!
